Working version of welcome email for Deno.

Shaka-n opened the pull request:
I used a simplified client to interact with the Mailgun API because I had trouble getting Deno's compatibility module to import the mailgun.js module. I was also not sure how to manually alter the event payload to test if the function fully works in AppWrite, but I have it working locally. Looking forward to making any necessary changes :)

TorstenDittmann commented on deno/welcome-email/index.js:
```diff
@@ -0,0 +1,19 @@
+import MailgunClient from "https://gist.githubusercontent.com/webdeb/837886746bf43e46effb269c68f0f5ce/raw/1eff4888e78ea1d8d2998c62284d1f7711ce205c/Mailgun.ts"
```
Instead of importing `https://gist.githubusercontent.com/webdeb/837886746bf43e46effb269c68f0f5ce/raw/1eff4888e78ea1d8d2998c62284d1f7711ce205c/Mailgun.ts`, you could put this file in the directory. This way we can make sure it is always available 🙂

Shaka-n commented:
Thanks Torsten! I could put that file in the directory, but I'm not sure what the fair use is around that. I figured it would be easier if I removed that external import and replaced it with a simple fetch request. Let me know if you disagree though!

TorstenDittmann commented:
> Thanks Torsten! I could put that file in the directory, but I'm not sure what the fair use is around that. I figured it would be easier if I removed that external import and replaced it with a simple fetch request. Let me know if you disagree though!

Both works fine with me 🙂

TorstenDittmann reviewed (changes requested):
Just remove the `deno/code.tar.gz` file and it is good to go! 🥳

Glide7 commented:
@TorstenDittmann how can we move with this? :)

TorstenDittmann commented:
@eldadfux needs to merge this. I thought I tagged him back then 🤐

Glide7 commented:
@eldadfux reminder for this :)

eldadfux commented:
🚀
